app/components/primer/alpha/button_marketing.pcss

```
@@ -0,0 +1,175 @@
+.btn-mktg {
+  position: relative;
+  z-index: 1;
+  display: inline-block;
+
+  /* stylelint-disable-next-line primer/spacing */
+  padding: 0.9rem 1.5rem 1.1rem;
+
+  /* stylelint-disable-next-line primer/typography */
+  font-size: 1rem;
+  font-weight: $font-weight-bold;
```

I think this can be replaced with

```suggestion
  font-weight: var(--base-text-weight-semibold, 600);
```

form [Primitives](https://primer.style/primitives/typography#weight).